Explorer/Assets/Scripts/ECS/StreamableLoading/GLTF/GltFastDownloadProvider.cs

```diff
@@ -56,24 +63,38 @@ public async Task<IDownload> RequestAsync(Uri uri)
             }
 
             uri = new Uri(tryGetContentUrlResult);
+            var commonArguments = new CommonArguments(URLAddress.FromString(GetUrl(uri)));
+
+            byte[] data = Array.Empty<byte>();
+            string error = string.Empty;
+            string text = string.Empty;
+            bool success = true;
```

Comment:
instead of being `true` by default, shouldn't the `success` value be bound to 
 `!string.IsNullOrEmpty(downloadHandler.error)` ? maybe in the `finally`.

Reply:
Good catch, the request might not throw an exception but still return a fail state, thanks!